internal/services/loadbalancer/loadbalancer_nat_rule_resource_test.go

```diff
@@ -165,6 +135,59 @@ func TestAccAzureRMLoadBalancerNatRule_updateMultipleRules(t *testing.T) {
 	})
 }
 
+func TestAccAzureRMLoadBalancerNatRule_mapToBackendAddressPool(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_lb_nat_rule", "test")
+	r := LoadBalancerNatRule{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.mapToBackendAddressPool(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep(),
+	})
+}
+
+func TestAccAzureRMLoadBalancerNatRule_multipleRuleMapToBackendAddressPool(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_lb_nat_rule", "test")
+	r := LoadBalancerNatRule{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.multipleRuleMapToBackendAddressPool(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep(),
+	})
+}
+
+// todo: currently, the nat rule cannot be reverted backed from V2 to V1, service team is fixing it now.
```

Comment:
Can we submit a issue in the azure-rest-api-specs repo and put the link here for better tracking this?

Reply:
sure, will do.